apps/examples/iotivity_simpleserver/Makefile

```diff
@@ -0,0 +1,186 @@
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+############################################################################
+
+-include $(TOPDIR)/.config
+-include $(TOPDIR)/Make.defs
+include $(APPDIR)/Make.defs
+
+# built-in application info
+
+APPNAME = simpleServer
+THREADEXEC = TASH_EXECMD_ASYNC
+
+# iotivitySimServ example
+IOTIVITY_RELEASE=${shell echo $(CONFIG_IOTIVITY_RELEASE_VERSION) | sed 's/"//g'}
+IOTIVITY_BASE_DIR = $(APPDIR)/../external/iotivity/iotivity_$(IOTIVITY_RELEASE)
+
+ASRCS =
+
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/resource/c_common
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/resource/csdk
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/resource/csdk/connectivity/api
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/resource/csdk/connectivity/common/inc
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/resource/csdk/logger/include
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/resource/csdk/security/include
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/resource/csdk/stack/include
+CFLAGS += -I$(IOTIVITY_BASE_DIR)/extlibs/mbedtls/mbedtls/include
+
+CFLAGS += -DWITH_POSIX -DIP_ADAPTER
+CFLAGS += -DWITH_BWT
+ifeq ($(CONFIG_IOTIVITY_ROUTING),"EP")
+CFLAGS += -DROUTING_EP
+else
+CFLAGS += -DROUTING_GW
+endif
+CFLAGS += -D__TINYARA__
+CFLAGS += -std=c99
+CFLAGS += -w
+CFLAGS += -D__WITH_DTLS__
+
+ifeq ($(CONFIG_EXAMPLES_IOTIVITY),y)
+CSRCS += port_tinyara.c
+else
+CSRCS += port_tinyara.c
+endif
+
+MAINSRC = simpleServer.c
+
+AOBJS = $(ASRCS:.S=$(OBJEXT))
+COBJS = $(CSRCS:.c=$(OBJEXT))
+MAINOBJ = $(MAINSRC:.c=$(OBJEXT))
+
+SRCS = $(ASRCS) $(CSRCS) $(MAINSRC)
+OBJS = $(AOBJS) $(COBJS)
+
+ifneq ($(CONFIG_BUILD_KERNEL),y)
+  OBJS += $(MAINOBJ)
+endif
+
+ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+  BIN = ..\..\libapps$(LIBEXT)
+else
+ifeq ($(WINTOOL),y)
+  BIN = ..\\..\\libapps$(LIBEXT)
+else
+  BIN = ../../libapps$(LIBEXT)
+endif
+endif
+
+ifeq ($(WINTOOL),y)
+  INSTALL_DIR = "${shell cygpath -w $(BIN_DIR)}"
+else
+  INSTALL_DIR = $(BIN_DIR)
+endif
+
+CONFIG_EXAMPLES_IOTIVITY_PROGNAME ?= iotivitySimServ$(EXEEXT)
+PROGNAME = $(CONFIG_EXAMPLES_IOTIVITY_PROGNAME)
+
+
+ROOTDEPPATH = --dep-path .
+
+# Common build
+
+VPATH =
+
+all: .built
+.PHONY: clean depend distclean preconfig
+
+$(AOBJS): %$(OBJEXT): %.S
+	$(call ASSEMBLE, $<, $@)
+
+$(COBJS) $(MAINOBJ): %$(OBJEXT): %.c
+	$(call COMPILE, $<, $@)
+
+.built: $(OBJS)
+	$(call ARCHIVE, $(BIN), $(OBJS))
+	@touch .built
+
+ifeq ($(CONFIG_BUILD_KERNEL),y)
+$(BIN_DIR)$(DELIM)$(PROGNAME): $(OBJS) $(MAINOBJ)
+	@echo "LD: $(PROGNAME)"
+	$(Q) $(LD) $(LDELFFLAGS) $(LDLIBPATH) -o $(INSTALL_DIR)$(DELIM)$(PROGNAME) $(ARCHCRT0OBJ) $(MAINOBJ) $(LDLIBS)
+	$(Q) $(NM) -u  $(INSTALL_DIR)$(DELIM)$(PROGNAME)
+
+install: $(BIN_DIR)$(DELIM)$(PROGNAME)
+
+else
+install:
+
+endif
+
+ifeq ($(CONFIG_BUILTIN_APPS)$(CONFIG_EXAMPLES_IOTIVITY),yy)
+$(BUILTIN_REGISTRY)$(DELIM)$(APPNAME)_main.bdat: $(DEPCONFIG) Makefile
+	$(call REGISTER,$(APPNAME),$(APPNAME)_main,$(THREADEXEC))
```

Comment:
Ok. it is hidden. Thank you.
But it makes another task to launch iotivity simple server. 
Do you need to make one more task? Is it a reason that you want to set specific stack size and priority?
Currently, TASH makes a task for ASYNC. but that makes another task again.